### DISCUSSION TITLE
How to disable a Switch?

### DISCUSSION THREAD

**Discussion by tpoxa** [tailwindlabs/headlessui#1956] [Help] [answered]
I want to disable a switch for a second while the request is processed to avoid double-clicking.
Can not find the "disable" attribute? 
I would appreciate any help. Same question here #1580 but maybe someones know some kind of trick. 

Thanks.

**Comment by shazha** [comment, marked as answer]
`disabled` attribute works

**Comment by tpoxa** [comment]
Oh my, indeed, thank you @shazha!
